content/developer/reference/extract_api.rst

```diff
@@ -473,49 +694,48 @@ Response
 ``id``
     |SAME_AS_PARSE|
 ``result``
-    ``status_code``
-        |STATUS_CODE|
+    ``status``
+        |SAME_AS_PARSE|
+    ``status_code`` (deprecated)
+        |SAME_AS_PARSE|
     ``status_msg``
-        |STATUS_MSG|
-
-=============  ==========================================================
- status_code       status_msg
-=============  ==========================================================
-0              Success
-12             Validation format is incorrect
-=============  ==========================================================
-
-.. rst-class:: setup doc-aside
-.. code-block:: text
-
-   {
-       "jsonrpc": string,
-       "id": string,
-       "result": {
-           "status_code": int,
-           "status_msg": string,
-       }
-   }
+        |SAME_AS_PARSE|
 
+========================  ===========================  ==========================================================
+status                    status_code (deprecated)     status_msg
+========================  ===========================  ==========================================================
+success                   0                            Success
+error_validation_format   12                           Validation format is incorrect
+========================  ===========================  ==========================================================
+
+.. code-block:: js
+
+    {
+        "jsonrpc": "2.0",
+        "id": string,
+        "result": {
+            "status": string,
+            // "status_code": int,  // deprecated
+            "status_msg": string,
+        }
+    }
 
 .. _webservices/extract_api/integration_testing:
 
 Integration Testing
 ===================
 
-You can test your integration by using *integration_token* as ``account_token`` in the :ref:`/parse <webservices/extract_api/invoice_parse>` request.
+You can test your integration by using *integration_token* as ``account_token`` in the
+:ref:`/parse </api/extract/__service__/1/parse>` request.
 
-Using this token put you in test mode and allows you to simulate the entire flow without really parsing a document and without being billed one credit for each successful invoice parsing.
+Using this token put you in test mode and allows you to simulate the entire flow without really parsing a document and
+without being billed one credit for each successful **invoice** parsing.
 
-The only technical differences in test mode is that the document you send is not parsed by the system and that the response you get from :ref:`/get_results <webservices/extract_api/invoice_get_results>`
-is a hard-coded one.
+The only technical differences in test mode is that the document you send is not parsed by the system and that the
+response you get from :ref:`/get_results </api/extract/__service__/1/get_results>` is a hard-coded one.
```

Comment:
This was just because of the ref... I can just edit what is visible since I used the same anchor for both routes.
```suggestion
response you get from :ref:`/get_result </api/extract/__service__/1/get_results>` is a hard-coded one.
```

Reply:
Ok, I see, then we might want to also change the anchor to `get_result`, no ?